tools/lsp/ui/draw-area.slint

```diff
@@ -96,7 +103,8 @@ export component DrawArea {
     callback select-at(/* x */ length, /* y */ length, /* enter_component? */ bool);
     callback select-behind(/* x */ length, /* y */ length, /* enter_component? */ bool, /* reverse */ bool);
     callback show-document(/* url */ string, /* line */ int, /* column */ int);
-    callback reselect(); // Reselect element e.g. after changing the window size (which may move the element)
+    callback reselect();
+    // Reselect element e.g. after changing the window size (which may move the element)
```

Comment:
I guess you got bitten by the formatter there. But comment usually applies to the line after